<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>teacher</class>
 <widget class="QMainWindow" name="teacher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>教师页面</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
font-size:24px;
font-family:sans-serif;
}
#centralwidget{   
border-image:url(:/image/7.jpg);
}
QFrame{
background:rgba(0,0,0,0);
border-radius:15px;
}
QPushButton{
background:#03a9f4;
color:#fff;
border-radius:15px;
}
QLineEdit{
border-radius:15px;
color:#03a9f4;
}
QLabel{
color:#fff;
background:transparent;
font-size:30px;
}
QComboBox{
	background:#03a9f4;
color:#fff;
border-radius:15px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>-10</y>
      <width>811</width>
      <height>524</height>
     </rect>
    </property>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>791</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>320</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>学号</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>姓名</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>成绩</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="lineEdit_findname">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>320</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>270</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>学号</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>年龄</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>成绩</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>270</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>排序</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>270</y>
       <width>317</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>降序</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>370</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>显示信息</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>440</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>修改信息</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>370</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>删除学生</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>440</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>添加学生</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
